hosts

```diff
@@ -36,6 +36,9 @@ cdh-web1.princeton.edu
 cdhweb_qa
 cdhweb_prod
 
+[geotaste]
+cdh-dev-sandbox1.princeton.edu
```

Comment:
Noting for myself that we still have to figure out how to split out host from app; we may need to add app-specific group vars. I think it's fine if we tackle this when we write a second playbook for the sandbox (probably / hopefully for simrisk)